GitHub-trainings/git-github-ttl-intro/template.qmd

```diff
@@ -0,0 +1,410 @@
+::: {.column width="70%"}
+Version control is not unique to Git. OneDrive, DropBox etc. also has version control
+
+Instead of saving the full file for each version (very inefficient), Git only saves the difference from the previous version (very efficient)
+
+Each saved line-by-line difference is called a "**_commit_**". This is the central building block in Git
+
+OneDrive, DropBox etc. saves full versions. Therefore, even if you have the most expensive enterprise subscription, old versions will eventually be permanently deleted
+:::
+
+::: {.column width="10%"}
+:::
+
+::: {.column width="10%"}
+<img src="img/network-graph-1.png"/>
+:::
+::::
+
+
+## Gives access to **all** historic versions
+
+:::: {.columns .v-center-container}
+::: {.column width="60%"}
+Since commits are so efficient, all historic versions of your project is stored on your computer without taking up significant disk space
+
+You can tell Git to show any version of your project's code folder - this is done without creating a chaos of multiple versions of each file
+
+Git use the commits to "_calculate_" any version you select to "**_check-out_**"
+:::
+
+::: {.column width="20%"}
+:::
+
+::: {.column width="10%"}
+<img src="img/network-graph-1.png"/>
+:::
+::::
+
+
+## GitHub is very clickable
+
+:::: {.columns .v-center-container}
+::: {.column width="70%"}
+On GitHub, each commit is given a unique URL that can be shared within team to make conversation about code edits very exact. 
+
+These links are permanent and will forever show the edits in, for example, "D"
+
+TODO: Image of commit on GitHub
+
+:::
+
+::: {.column width="10%"}
+:::
+
+::: {.column width="10%"}
+<img src="img/network-graph-1.png"/>
+:::
+::::
+
+## A commit is a snap shot of the full project
+
+:::: {.columns .v-center-container}
+::: {.column width="50%"}
+
+A commit is on the project level. This means, a commit includes the difference in all project files relative the previous commit
+
+If you check-out an old commit, Git restores all files in the project folder to that version
+:::
+
+::: {.column width="50%"}
+TODO: Image of commit for multiple files
+:::
+::::
+
+
+## GitHub is very clickable
+
+:::: {.columns .v-center-container}
+::: {.column width="70%"}
+
+It is also [possible](https://stackoverflow.com/a/63940108/6889416) to show the sum of the differences in multiple commits
+
+TODO: Image of compare of multiple commits on GitHub
+:::
+
+::: {.column width="10%"}
+:::
+
+::: {.column width="10%"}
+<img src="img/network-graph-1.png"/>
+:::
+::::
+
+# Git Clients
+
+## GitHub Desktop
+
+:::: {.columns .v-center-container}
+::: {.column width="50%"}
+
+A Git Client is a software that allows you to package your edits into commits. Many alternatives exists.
+
+We strongly recommend everyone using a graphic Git Client such as [GitHub Desktop](https://desktop.github.com/). Unless you are a computer scientists do not use the CLI tool until you are an advanced Git user.
```

Comment:
I suggest mentioning here when presenting (not necessarily in slides) that it doesn't need admin approval/rights for installation in WB computers or any computer